nucliadb_models/nucliadb_models/search.py

```diff
@@ -341,16 +341,43 @@ def from_message(cls: Type[_T], message: PBShards) -> _T:
         return cls(**as_dict)
 
 
+class ParamDefaults:
+    query = dict(default="", title="Query", description="The query to search for")
+    advanced_query = dict(
+        default=None,
+        title="Advanced query",
+        description="An advanced query to search for. Follows the tantivy query parser syntax: https://docs.rs/tantivy/latest/tantivy/query/struct.QueryParser.html",  # noqa: E501
+    )
+    fields = dict(
+        default=[], title="Fields", description="The list of fields to search in"
+    )
+    filters = dict(
+        default=[],
+        title="Filters",
+        description="The list of filters to apply. The filters follow the syntax defined in https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md#filters-and-facets",  # noqa: E501
```

Comment:
Let's update these doc links to reference docs.nuclia.dev instead

Reply:
done!